alpha/alpha-server/src/main/resources/schema-mysql.sql

```diff
@@ -116,8 +116,8 @@ CREATE TABLE IF NOT EXISTS tcc_tx_event (
 
 CREATE TABLE IF NOT EXISTS master_lock (
   serviceName varchar(36) not NULL,
-  expireTime timestamp(3) not NULL,
-  lockedTime timestamp(3) not NULL,
+  expireTime datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
It looks like the expireTime should be NULL if no timeout setting. @WillemJiang  can you confirm it ?

Reply:
if expireTime can be null, the sql should change to:
```SQL
CREATE TABLE IF NOT EXISTS master_lock (
  serviceName varchar(36) not NULL,
  expireTime datetime,
  lockedTime datetime not NULL DEFAULT CURRENT_TIMESTAMP,
  instanceId  varchar(255) not NULL,
  PRIMARY KEY (serviceName)
) ENGINE=InnoDB DEFAULT CHARSET=utf8; 
```

Reply:
I have no problem testing in my local, it may be that your MySQL has set NO_ZEROR_DATE, you can check the following through the following command line

```bash
show variables like 'sql_mode'
```
Of course, I agree with this modification.

Reply:
If there  is no side effect of using the CURRENT_TIMESTAMP ad the default value. We could just accept the PR.